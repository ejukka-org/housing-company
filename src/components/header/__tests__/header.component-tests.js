import React from 'react';
import Header from '../header.component';
import renderer from 'react-test-renderer';

test('Renders Header component', () => {
    const component = renderer.create(
        <Header />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
