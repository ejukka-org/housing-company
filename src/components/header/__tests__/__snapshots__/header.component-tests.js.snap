// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders SignIn component 1`] = `
"<div class=\\"container\\">
  <nav class=\\"navbar navbar-expand-lg navbar-dark bg-dark\\"><a href=\\"/\\" class=\\"navbar-brand\\">Vihdin Mäkirinne 2</a><button aria-controls=\\"responsive-navbar-nav\\" type=\\"button\\" aria-label=\\"Toggle navigation\\" class=\\"navbar-toggler collapsed\\"><span class=\\"navbar-toggler-icon\\"></span></button>
    <div class=\\"navbar-collapse collapse\\" id=\\"responsive-navbar-nav\\">
      <div class=\\"mr-auto navbar-nav\\"><a href=\\"/apartments\\" data-rb-event-key=\\"/apartments\\" class=\\"nav-link\\">Asunnot</a><a href=\\"/history\\" data-rb-event-key=\\"/history\\" class=\\"nav-link\\">historia</a>
        <div class=\\"dropdown nav-item\\"><a aria-haspopup=\\"true\\" aria-expanded=\\"false\\" href=\\"#\\" class=\\"dropdown-toggle nav-link\\" role=\\"button\\">Linkkejä</a></div>
      </div><a href=\\"/admin\\" class=\\"nav-link\\">admin</a>
      <div class=\\"navbar-nav\\"><a href=\\"/about\\" data-rb-event-key=\\"/about\\" class=\\"nav-link\\">Lisätietoa</a><button type=\\"button\\" class=\\"btn btn-primary\\">SIGN IN</button></div>
    </div>
  </nav>
</div>"
`;
